<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>azizwindow</class>
 <widget class="QMainWindow" name="azizwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>1012</width>
      <height>543</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_21">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_19">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_18">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_14"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_imprimer">
            <property name="text">
             <string>imprimer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="tableViewAffichuer"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_actualiser">
            <property name="text">
             <string>Actualiser</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_44">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_43">
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>CRUD</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_42">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_17">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_41">
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_3">
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_2">
                       <item>
                        <widget class="QLabel" name="label">
                         <property name="text">
                          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ID&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="lineEdit_ID"/>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_12">
                       <item>
                        <widget class="QLabel" name="label_12">
                         <property name="text">
                          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Nom&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="lineEdit_Nom"/>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_3">
                       <item>
                        <widget class="QLabel" name="label_2">
                         <property name="text">
                          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Prenom&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="lineEdit_Prenom"/>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_9">
                       <item>
                        <widget class="QLabel" name="label_10">
                         <property name="text">
                          <string>Sexe : </string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <layout class="QVBoxLayout" name="verticalLayout">
                         <item>
                          <widget class="QRadioButton" name="Homme">
                           <property name="text">
                            <string>Homme</string>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QRadioButton" name="Femme">
                           <property name="text">
                            <string>Femme</string>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_7">
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_7">
                       <item>
                        <widget class="QLabel" name="label_7">
                         <property name="text">
                          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Numéro de téléphone&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="lineEdit_TEL"/>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_4">
                       <item>
                        <widget class="QLabel" name="label_4">
                         <property name="text">
                          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;CIN&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="lineEdit_Cin"/>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_5">
                       <item>
                        <widget class="QLabel" name="label_6">
                         <property name="text">
                          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;EMAIL: &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_3">
                         <property name="text">
                          <string/>
                         </property>
                         <property name="pixmap">
                          <pixmap>../../../tech-wallpaper-4k-for-mobile.png</pixmap>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="lineEdit_Email"/>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_39">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_6">
                     <item>
                      <widget class="QPushButton" name="pushButton_Ajouter">
                       <property name="text">
                        <string>Ajouter</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="pushButton_Modifer">
                       <property name="text">
                        <string>Modifier</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout">
                     <item>
                      <widget class="QLabel" name="label_9">
                       <property name="text">
                        <string>id: </string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="lineEdit_ID_Supression"/>
                     </item>
                     <item>
                      <widget class="QPushButton" name="pushButton_Suppression">
                       <property name="text">
                        <string>supprimer</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Type: </string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QRadioButton" name="Fidele">
                  <property name="text">
                   <string>Fidéle</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="Passage">
                  <property name="text">
                   <string>Passagé</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_20">
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Recherche et Tri</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_12">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <widget class="QLabel" name="label_13">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_nom_chercher"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_Rechercher">
                   <property name="text">
                    <string>Rechercher</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_38">
                 <item>
                  <widget class="QPushButton" name="pushButton_trier">
                   <property name="text">
                    <string>Trier par id</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_trier_nom">
                   <property name="text">
                    <string>Trier par nom</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Génération PDF</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_13">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_23">
                 <item>
                  <widget class="QLabel" name="label_34">
                   <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ID&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_ID_PDF"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_24">
                 <item>
                  <widget class="QPushButton" name="pushButtonGenerePDF">
                   <property name="text">
                    <string>Générer</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_AfficherPDF">
                   <property name="text">
                    <string>Afficher</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_stat">
        <property name="text">
         <string>stat</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_sombre">
        <property name="text">
         <string>sombre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_blanc">
        <property name="text">
         <string>blanc</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Historique CRUD</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_16">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_40">
        <item>
         <widget class="QTableView" name="tableViewHistorique"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonActualiserHistorique">
          <property name="text">
           <string>Actualiser</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QPushButton" name="ouvrir">
      <property name="geometry">
       <rect>
        <x>751</x>
        <y>190</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>ouvrir caisse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>190</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>alarme</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>491</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_caisse">
      <property name="geometry">
       <rect>
        <x>641</x>
        <y>193</y>
        <width>104</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>270</y>
        <width>491</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
